cloud-core/src/main/java/cloud/commandframework/execution/CommandExecutionCoordinator.java

```diff
@@ -115,10 +116,19 @@ private SimpleCoordinator(final @NonNull CommandTree<C> commandTree) {
                     completableFuture.completeExceptionally(pair.getSecond());
                 } else {
                     final Command<C> command = Objects.requireNonNull(pair.getFirst());
+                    CommandExecutionException commandExecutionException = null;
                     if (this.getCommandTree().getCommandManager().postprocessContext(commandContext, command) == State.ACCEPTED) {
-                        command.getCommandExecutionHandler().execute(commandContext);
+                        try {
+                            command.getCommandExecutionHandler().execute(commandContext);
+                        } catch (Throwable throwable) {
```

Comment:
Bad! Error is special!

Reply:
No its not.

Reply:
`Error`s should not be caught in reasonable applications, as the javadoc of it says: https://docs.oracle.com/javase/8/docs/api/java/lang/Error.html